dev-tools/mage/gotest.go

```diff
@@ -28,12 +27,10 @@ import (
 	"os/exec"
 	"path"
 	"path/filepath"
-	"runtime"
 	"sort"
 	"strings"
 	"time"
 
-	"github.com/jstemmer/go-junit-report/formatter"
 	"github.com/jstemmer/go-junit-report/parser"
```

Comment:
Is it still needed?

Reply:
nice catch. It should not be. Let me check if there is some other dependency that I might have missed

Reply:
Found it. We used to have a GoTestSummary printer which parses the output in order to print a "rich" summary. With the switch to `gotestsum` it is not used anymore. The gotestsum tool prints a short summary, but it is not as rich as the old report. But all in all we don't really need it, as all details are already written to stdout. I'd rather reduce the amount of customization we have in our build tools if possible.